src/plots/mapbox/layout_attributes.js

```diff
@@ -48,10 +48,16 @@ var attrs = module.exports = overrideAll({
         dflt: constants.styleValueDflt,
         role: 'style',
         description: [
-            'Sets the Mapbox map style.',
-            'Either input one of the default Mapbox style names or the URL to a custom style',
-            'or a valid Mapbox style JSON.',
-            'From OpenStreetMap raster tiles, use *open-street-map*.'
+            'Sets the Mapbox base map style.',
```

Comment:
I would remove the word "Mapbox" here as it's ambiguous if we're talking about Mapbox-provided styles or not.

Reply:
Good call. Done in https://github.com/plotly/plotly.js/pull/4035/commits/ca762e96d9f8ea77966cfe4918c292e48661f3b0